egs/swbd/run.sh

```diff
@@ -10,6 +10,15 @@ num_word=20000
 lm_dir="data/lm/"
 arpa_dir="data/arpa/"
 
+max_memory='--max-memory=10G'
+# If you do not want to set memory limitation for "sort", you can use
+#max_memory=
+# Choices for the max-memory can be:
+# 1) integer + 'K', 'M', 'G', ... 
+# 2) integer + 'b', meaning using default unit byte and no multiplication
+# 3) integer + '%', meaning a percentage of memory
+# 4) integer, default unit is byte
```

Comment:
Hi @keli78, I read manual page of sort(1) just now, it says the default unit of --buffer-size= is 'K' not 'b'. Please correct these lines of comment.